core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UserStoreConfigConstants.java

```diff
@@ -153,4 +153,8 @@
             + " inorder to establish the connection after couple of failure attempts.";
     public static final int DEFAULT_CONNECTION_RETRY_DELAY_IN_MILLISECONDS = 120000;
     public static final String OBJECT_GUID = "objectGuid";
+
+    // Property for specify case sensitivity for LDAP.
+    public static final String CASE_SENSITIVE_USERNAME = "CaseInsensitiveUsername";
```

Comment:
The value looks contrary to the variable name.

Reply:
The same constant is mentioned in [1]. Since it is in the JDBC constants, it is not the best practice to use it in AD/LDAP constants. Based on that created a common one. 

[1] https://github.com/wso2/carbon-kernel/blob/eaec36ab6e8f5b70c0d6dd40a32e02f8c5b87130/core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/caseinsensitive/JDBCCaseInsensitiveConstants.java#L250

Reply:
Even in that case, the variable name and the value sound contrary to one another. Since we are adding it anew here, better if we correct it IMO. @madurangasiriwardena any input?